Assets/Code/src/Runtime/Match/Systems/CollisionEventsReceiverAuthoring.cs.meta

```diff
@@ -0,0 +1,15 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:Assets/Code/src/Runtime/Match/Systems/CollisionEventsReceiverAuthoring.cs.meta
+guid: c5e412e42d51b1440b6b727fe354ae42
+=======
+guid: 853708c8afb59ca449321995a4d5059b
+>>>>>>> 4b7c8e15c8770b3d0eda3a51e87edef569046d33:Assets/Code/src/Runtime/Authoring/HitboxAuthoring.cs.meta
```

Comment:
This is a merge conflict. Please resolve this.  It looks like the first part is likely what you want instead of the second. Please verify that this doesn't break anything once you resolve the error.